package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
-    "lint": "tslint --project tsconfig.json --type-check && sass-lint -v -q",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx && sass-lint -v -q",
```

Comment:
Dev dependencies usually get added to the path when run as a script so eslint on its own should work

Reply:
of course, a typo :)